.devcontainer/devcontainer.json

```diff
@@ -18,14 +18,14 @@
   // Do not run as root. See https://aka.ms/vscode-remote/containers/non-root.
   "remoteUser": "azure-sdk-for-cpp",
   "containerEnv": {
-    "AZURE_CLIENT_ID": "",
-    "AZURE_CLIENT_SECRET": "",
-    "AZURE_KEYVAULT_HSM_URL": "",
-    "AZURE_KEYVAULT_URL": "",
+    "AZURE_CLIENT_ID": "272b264a-2ef7-4491-b833-344e673e3bfd",
+    "AZURE_CLIENT_SECRET": "45d480eb-6b2e-46f9-9708-b39de1977b0e",
```

Comment:
hope this is not real :P.  We don't want real values pushed to main.

If it is real, you might want to rotate the secret now, as it is already in git story